Commands/GiveItem.cs

```diff
@@ -0,0 +1,103 @@
+﻿using ProjectM;
+using Unity.Entities;
+using VampireCommandFramework;
+using System.Collections.Generic;
+
+namespace CommunityCommands.Commands
+{
+    internal static class GiveItem
+    {
+        private static Dictionary<string, List<Item>> equipmentSets = Database.GetSetDatabase();
+
+        [Command("give", "g", description: "Give a specific item or set of items to yourself", adminOnly: true)]
+        public static void GiveItemCommand(ChatCommandContext ctx, string type, string name)
+        {
+            if (string.IsNullOrEmpty(type) || string.IsNullOrEmpty(name))
+            {
+                ctx.Reply("Please provide valid arguments for <color=#ff0000>type</color> and <color=#ff0000>name</color>.");
+                return;
+            }
+
+            type = type.ToLower();
+
+            if (type == "set")
+            {
+                GiveEquipmentSet(ctx, name);
+            }
+            else if (type == "item")
+            {
+                GiveSingleItem(ctx, name);
+            }
+            else
+            {
+                ctx.Reply($"Invalid type <color=#ff0000>{type}</color>. Valid types are <color=#00ff00>set</color> and <color=#00ff00>item</color>.");
+            }
+        }

```

Comment:
```suggestion
internal enum GiveType {Item, Set};
        
        [Command("give", "g", description: "Give a specific item or set of items to yourself", adminOnly: true)]
        public static void GiveItemCommand(ChatCommandContext ctx, GiveType type, string name)
        {
            if (type == GiveType.Set)
            {
                GiveEquipmentSet(ctx, name);
            }
            else if (type == GiveType.Item)
            {
                GiveSingleItem(ctx, name);
            }
        }
```

Whenever possible try to get parsing and argument validation out of the commands and into types. Ultimately this will let VCF reflect better usage.